airflow/executors/base_executor.py

```diff
@@ -479,3 +479,12 @@ def send_callback(self, request: CallbackRequest) -> None:
         if not self.callback_sink:
             raise ValueError("Callback sink is not ready.")
         self.callback_sink.send(request)
+
+    @staticmethod
+    def get_cli_commands() -> list:
```

Comment:
I guess list[GroupCommand] or list[GroupCommand]. This is part of the public API, I think not specifying this leaves a lot of room for interpretation, what it could be. I think it only makes sense that executor adds the whole "Group" rather than single command (very similar as in case of config contributed for provider where we decided only whole section can be contributed by provider).